pkg/handlers/action.go

```diff
@@ -73,7 +73,19 @@ func (h ActionHandler) Handle() http.HandlerFunc {
 				return
 			}
 
+			intakeStatus := action.ActionType
```

Comment:
If I'm reading this right, this just returns a body that is a bare string value, like `CORRECT` (using a made-up thingy for example).
Usually, our response object are actual JSON bodies, more along the lines of `{"actionType":"CORRECT"}`...

Reply:
Isaac, really good catch because I didn't realize that there was a difference between status and actions in terms of actual data, this solution may? not work in some cases and is not very future proof since it ties status to actions in a very hidden way.

Im going to look more into this and potentially rethink this solution, maybe some kind of mapping from actions to statuses?

Reply:
And I'll be sure to put whatever I end up doing into actual JSON bodies rather then bare strings